sde_collections/tasks.py

```diff
@@ -215,6 +216,14 @@ def fetch_and_replace_full_text(collection_id, server_name):
             collection.reindexing_status = ReindexingStatusChoices.REINDEXING_READY_FOR_CURATION
             collection.save()
 
+        curated_count = collection.count_curated_urls()
+        dump_count = collection.count_dump_urls()
```

Comment:
This line should be executed prior to migrate dump to delta. It is advertised to the end user as "num urls successfully imported" or something descriptive of that effect.

Reply:
urls imported